docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -2,3 +2,349 @@
+
+The workflow uses the PBE exchange-correlation functional with suitable pseudopotentials and energy cutoffs from the `SSSP library version 1.1 <https://www.materialscloud.org/discover/sssp/table/efficiency>`_.
+
+.. K-point mesh is selected to have a minimum k-point density of 0.2 Å-1
+   A Marzari-Vanderbilt smearing of 0.02 Ry is used for the electronic occupations
+
+The workflow should take ~10 minutes on your virtual machine.
+You may notice that ``verdi process list`` now shows more than one entry:
+
+.. code-block:: console
+
+    $ verdi process list
+      PK  Created    Process label             Process State    Process status
+    ----  ---------  ------------------------  ---------------  --------------------------------------
+     218  6s ago     PwBandStructureWorkChain  ⏵ Waiting        Waiting for child processes: 233
+     233  4s ago     PwBandsWorkChain          ⏵ Waiting        Waiting for child processes: 235
+     235  3s ago     PwRelaxWorkChain          ⏵ Waiting        Waiting for child processes: 238
+     238  2s ago     PwBaseWorkChain           ⏵ Waiting        Waiting for child processes: 244
+     244  1s ago     PwCalculation             ⏵ Waiting        Monitoring scheduler: job state QUEUED
+
+    Total results: 5
+
+    Info: last time an entry changed state: 0s ago (at 12:45:40 on 2020-11-25)
+
+While you wait for the workflow to complete, let's start exploring its provenance.
+
+The full provenance graph obtained from ``verdi node graph generate`` will already be rather complex (you can try!), so let's try browsing the provenance interactively instead.
+
+Start the AiiDA REST API:
+
+.. code-block:: console
+
+    $ verdi restapi
+
+and open the |provenance browser| (from the browser inside the virtual machine).
+
+.. |provenance browser| raw:: html
+
+    <a href="https://www.materialscloud.org/explore/ownrestapi?base_url=http://127.0.0.1:5000/api/v4" target="_blank">Materials Cloud provenance browser</a>
+
+.. note::
+
+    The provenance browser is a Javascript application that connects to the AiiDA REST API.
+    Your data never leaves your computer.
+
+.. some general comment on importance of the graph?
+.. a sentence on how to continue from here
+
+Browse your AiiDA database:
+
+    * Start by finding your `Quantum ESPRESSO`_ calculation (the type of node is called a ``CalcJobNode`` in AiiDA, since it is run as a job on a scheduler).
+    * Select ``Calculations`` in the left menu to filter for calculations only.
+    * Inspect the raw inputs and outputs of the calculation, and use the provenance browser to explore the input and output nodes of the calculation and the whole provenance of your simulations.
+
+.. note::
+
+    When perfoming calculations for a publication, you can export your provenance graph using ``verdi export create`` and upload it to the `Materials Cloud Archive`_, enabling your peers to explore the provenance of your calculations online.
```

Comment:
"When perfoming calculations for a publication, you can export your provenance graph" sounds a bit to me like you are exporting the image of the node connections. Maybe I would add a clariffication here:

```suggestion
    When perfoming calculations for a publication, you can export your provenance graph (meaning all the content of the nodes and their connections) into an archive file using ``verdi export create``, and then upload it to the `Materials Cloud Archive`_, enabling your peers to explore the provenance of your calculations online.
```